brainstormforce/ultimate-addons-for-gutenberg
# fix: Inspector Tabs Rendering issues when not Consistent Tabs.

### Description
Inspector Tabs Rendering issues when not Consistent Tabs.

### Types of changes
 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
 - Switch to next-release.
 - Drag Drop Pricelist Block
 - Select Pricelist Child
 - You will see null written at the bottom. (Issue #1)
 - Select Parent now.
 - Click on Style Tab.
 - You will see all the settings above Tabs (Issue #2)
 - Switch to this Branch.
 - Above issues should be fixed.

### Checklist:
